### PR TITLE
Added fake google-services.json file to enable nightly build

### PR DESCRIPTION
**Pull request by ggfan**

### CODE REVIEW

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "_comment": "Fake service file, replace with your own versiona",
```
Comment (codingjeremy) [inline ClassyTaxiAppKotlin/app/src/debug/google-services.json:2]: version

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "project_info": {
+    "project_number": "",
```
Comment (codingjeremy) [inline ClassyTaxiAppKotlin/app/src/debug/google-services.json:4]: Should we do...
    "project_number": "TODO: Add your project number here",

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    "project_id": ""
```
Comment (codingjeremy) [inline ClassyTaxiAppKotlin/app/src/debug/google-services.json:5]: Should we do...
    "project_id": "TODO: Add your project id here",

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1526371418:android:2871699",
```
Comment (codingjeremy) [inline ClassyTaxiAppKotlin/app/src/debug/google-services.json:10]: Should we do...
    "mobilesdk_app_id": "1:1526371418:android:some_number_you_fill__in",

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.example.subscriptions",
+            "certificate_hash": ""
+          }
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": ""
```
Comment (codingjeremy) [inline ClassyTaxiAppKotlin/app/src/debug/google-services.json:31]: Should we do...
    "current_key": "TODO:ADD_KEY",

File: ClassyTaxiJava/app/src/debug/google-services.json [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1526371418:android:2871699",
+        "android_client_info": {
+          "package_name": "com.sample.android.classytaxijava"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "",
```
Comment (codingjeremy) [inline ClassyTaxiJava/app/src/debug/google-services.json:17]: Add TODO like above?

File: ClassyTaxiJava/app/src/debug/google-services.json [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+          "client_type": 3
+        },
+        {
+          "client_id": "",
```
Comment (codingjeremy) [inline ClassyTaxiJava/app/src/debug/google-services.json:21]: Add TODO like above?

File: ClassyTaxiJava/app/src/debug/google-services.json [hunk 8]
```diff
@@ -0,0 +1,53 @@
+{
+  "_comment": "fake service file, replace with your own one",
+  "project_info": {
+    "project_number": "",
+    "project_id": ""
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:1526371418:android:2871699",
```
Comment (codingjeremy) [inline ClassyTaxiJava/app/src/debug/google-services.json:10]: Add TODO like above?

File: ClassyTaxiJava/app/src/debug/google-services.json [hunk 9]
```diff
@@ -0,0 +1,53 @@
+{
+  "_comment": "fake service file, replace with your own one",
+  "project_info": {
+    "project_number": "",
+    "project_id": ""
```
Comment (codingjeremy) [inline ClassyTaxiJava/app/src/debug/google-services.json:5]: Add TODO like above?

File: ClassyTaxiJava/app/src/debug/google-services.json [hunk 10]
```diff
@@ -0,0 +1,53 @@
+{
+  "_comment": "fake service file, replace with your own one",
+  "project_info": {
+    "project_number": "",
```
Comment (codingjeremy) [inline ClassyTaxiJava/app/src/debug/google-services.json:4]: Add TODO like above?

**Review by codingjeremy — COMMENTED** [review]
Some of the comments you might want to ignore if they are needed as they are now to be parsed (and not raise a build error), but it might be helpful (if they don't cause build problems) to make them TODO strings.

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json (hunk 4 above)
Comment (ggfan) [reply ClassyTaxiAppKotlin/app/src/debug/google-services.json]: Good point, replace it with the default values from the documentation:
  https://firebase.google.com/docs/projects/learn-more

File: ClassyTaxiAppKotlin/app/src/debug/google-services.json (hunk 5 above)
Comment (ggfan) [reply ClassyTaxiAppKotlin/app/src/debug/google-services.json]: done

File: ClassyTaxiJava/app/src/debug/google-services.json (hunk 10 above)
Comment (ggfan) [reply ClassyTaxiJava/app/src/debug/google-services.json]: done

File: ClassyTaxiJava/app/src/debug/google-services.json (hunk 9 above)
Comment (ggfan) [reply ClassyTaxiJava/app/src/debug/google-services.json]: done

File: ClassyTaxiJava/app/src/debug/google-services.json (hunk 6 above)
Comment (ggfan) [reply ClassyTaxiJava/app/src/debug/google-services.json:17]: done

File: ClassyTaxiJava/app/src/debug/google-services.json (hunk 7 above)
Comment (ggfan) [reply ClassyTaxiJava/app/src/debug/google-services.json:21]: done

**Review by ggfan — COMMENTED** [review]
addressed the reviews, please take a look again, thank you!

**Review by codingjeremy — APPROVED** [review]

**Pull request merged**